public/locales/en/user.json

```diff
@@ -55,7 +55,7 @@
     "title": "Favorites"
   },
   "login": {
-    "field_required": ""
```

Comment:
You can add English translation in your original PR.

Reply:
Original PR is already merged so doing it here.